packages/chakra-ui/src/Skeleton/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { BoxProps } from "../Box";
+
+export interface ISkeleton {
+  /**
+   * The color at the animation start
+   */
+  colorStart?: string;
+  /**
+   * The color at the animation end
+   */
+  colorEnd?: string;
+}
+
+export type SkeletonProps = ISkeleton & BoxProps;
+
+declare const Badge: React.FC<SkeletonProps>;
```

Comment:
Thank you for this PR. It looks great.

I noticed an error with `useTheme` when trying it in CodeSandbox. Not sure what the issue it.

In this line and below, you need to export `Skeleton` not `Badge`, I guess you missed that 😅

Reply:
oops, didn't notice